CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.12.5
+
+**Released**: 2021.05.06
+
+**Commit Delta**: [Change from 0.12.5 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.0..0.12.5)
```

Comment:
"0.12.0..0.12.5" -> "0.12.4..0.12.5"

Reply:
Thank you! Missed that :(

Reply:
It's rather rich of me to point out an error in a CHANGELOG.md file given how many mistakes I've made with that file in various repos.